sdk/nodejs/ad/getApplication.ts

```diff
@@ -7,7 +7,9 @@ import * as utilities from "../utilities";
 /**
  * > This content is derived from https://github.com/terraform-providers/terraform-provider-azurerm/blob/master/website/docs/d/azuread_application.html.markdown.
  */
+/** @deprecated The azure/ad module has moved to a dedicated Pulumi azuread package */
 export function getApplication(args?: GetApplicationArgs, opts?: pulumi.InvokeOptions): Promise<GetApplicationResult> & GetApplicationResult {
+	pulumi.log.warn("getApplication is deprecated: The azure/ad module has moved to a dedicated Pulumi azuread package")
```

Comment:
wonky indentation.

Reply:
I can't see how this one is wonky... the code looks like this in the SDK

```
/**
 * > This content is derived from https://github.com/terraform-providers/terraform-provider-azurerm/blob/master/website/docs/d/azuread_application.html.markdown.
 */
/** @deprecated The azure/ad module has moved to a dedicated Pulumi azuread package */
export function getApplication(args?: GetApplicationArgs, opts?: pulumi.InvokeOptions): Promise<GetApplicationResult> & GetApplicationResult {
	pulumi.log.warn("getApplication is deprecated: The azure/ad module has moved to a dedicated Pulumi azuread package")
    args = args || {};
    if (!opts) {
        opts = {}
    }

    if (!opts.version) {
        opts.version = utilities.getVersion();
    }
    const promise: Promise<GetApplicationResult> = pulumi.runtime.invoke("azure:ad/getApplication:getApplication", {
        "name": args.name,
        "objectId": args.objectId,
    }, opts);

    return pulumi.utils.liftProperties(promise, opts);
}
```

is this not correct?

Reply:
the indentation is off.

```typescript
	pulumi.log.warn("getApplication is deprecated: The azure/ad module has moved to a dedicated Pulumi azuread package")
    args = args || {};
    if (!opts) {
        opts = {}
    }
```

should be

```typescript
    pulumi.log.warn("getApplication is deprecated: The azure/ad module has moved to a dedicated Pulumi azuread package")
    args = args || {};
    if (!opts) {
        opts = {}
    }
```

Reply:
i think you're using tabs instead of spaces btw.

Reply:
Oh I see! fixing this up now as well